server/webapp/WEB-INF/rails/webpack/views/pages/users.tsx

```diff
@@ -94,7 +95,19 @@ export class UsersPage extends Page<null, State> {
         users: usersToDelete.userNamesOfSelectedUsers()
       };
 
-      this.bulkUserDelete(vnode, json);
+      new DeleteUserConfirmModal(json,
+                                 (msg) => {
```

Comment:
Instead of passing callbacks, can we just call `onDelete` in `ondelete` callback of `DeleteConfirmModal`?